packages/component-library/src/components/graphs/mt-graph/mt-graph.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <apexchart
+    :type="type"
+    :series="series"
+    :options="chartOptions"
+    :width="width"
+    :height="height"
+    :class="`mt-graph-${type}`"
+    data-testid="mt-graph"
+  />
+</template>
+
+<script setup lang="ts">
+import apexchart from "vue3-apexcharts";
+import type { ApexOptions } from "apexcharts";
+import { getDefaultOptions } from "./mt-graph-default-options";
+import { deepCopyObject, deepMergeObjects } from "@/utils/object";
+import { defineProps, computed } from "vue";
+
+export type GraphOptions = ApexOptions;
+
+const props = defineProps<{
+  series: any[];
+  options?: GraphOptions;
+  type?: string;
+  width?: string | number;
+  height?: string | number;
+}>();
+
+const chartOptions = computed<ApexOptions>(() => {
+  const type = props.type ?? "area";
+  const options = props.options;
+  const defaultOptions = getDefaultOptions(type);
+  const defaultOptionsCopy = deepCopyObject(defaultOptions);
```

Comment:
You don't need to create a copy anymore because `getDefaultOptions` always creates a new object. Btw. this was the intention behind my suggestion :)